lib/twilio_mock/mocker.rb

```diff
@@ -3,31 +3,44 @@
 require_relative 'number_generator'
 require 'ostruct'
 require_relative 'twilify'
+require 'active_support'
+require 'active_support/core_ext/object/to_query'
```

Comment:
Nice, I missed this as I used rails, do you think is worth adding this dependency for this code? Maybe we can put the code here:
```ruby
class Hash
 def to_query(namespace = nil)
    collect do |key, value|
      unless (value.is_a?(Hash) || value.is_a?(Array)) && value.empty?
        value.to_query(namespace ? "#{namespace}[#{key}]" : key)
      end
    end.compact.sort! * '&'
  end
end

require "cgi"

class Object
 def to_query(key)
    "#{CGI.escape(key.to_param)}=#{CGI.escape(to_param.to_s)}"
  end
end

```

You can forget about this, I will do the refactor myself after merging this